apps/meteor/client/providers/CallProvider/CallProvider.tsx

```diff
@@ -54,10 +57,29 @@ export const CallProvider: FC = ({ children }) => {
 
 	const [queueCounter, setQueueCounter] = useState(0);
 	const [queueName, setQueueName] = useState('');
+	const [roomInfo, setRoomInfo] = useState<{ v: { token?: string }; rid: string }>();
+
+	const closeRoom = useCallback(
+		async (data): Promise<void> => {
+			roomInfo &&
+				(await voipCloseRoomEndpoint({
+					rid: roomInfo.rid,
+					token: roomInfo.v.token || '',
+					options: { comment: data?.comment, tags: data?.tags },
+				}));
+			homeRoute.push({});
+
+			const queueAggregator = result?.voipClient?.getAggregator();
```

Comment:
there is no need to `?` for `result?`and `data?` or am I wrong?

Reply:
on result you are right! data is a different approach... Let me upgrade this PR, better things will come!